Search/Converter/ConverterManagerInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+ * This file is part of the MassiveSearchBundle
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Search\Converter;
+
+/**
+ * Interface for converter manager.
+ */
+interface ConverterManagerInterface
+{
+    /**
+     * Converts value from a source format to a target format.s.
+     *
+     * @param mixed $value
+     * @param string $from source format.
+     *
+     * @return mixed
+     */
+    public function convert($value, $from);
+
+    /**
+     * Returns true if a converter exists which converts from source to target format.
+     *
+     * @param string $from source format.
+     *
+     * @return bool
+     */
+    public function hasConverter($from);
```

Comment:
Where's the target here? It should be here according to the comment above. Same question for the other method.